### PR TITLE
Add document status to guides, templates

### PR DESCRIPTION
**Pull request by csarven**
This PR adds document status (modified and reason) to documents pertaining to the Solid CG. It follows a 2023-10-10 Solid CG meeting actions with the goal to clean-up / update references to CG's work. A related change was also made by https://github.com/solid/process/pull/331 .

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+<details open="">
+  <summary>Document Status</summary>
+  <dl>
+    <dt>Modified</dt>
+    <dd>2024-01-10</dd>
+    <dt>Reason</dt>
+    <dd>This document is no longer effective and is no longer being updated here. For the latest information on the <a href="https://www.w3.org/groups/cg/solid/">W3C Solid Community Group</a>, please refer to its <a href="https://github.com/w3c-cg/solid/">repository</a>, <a href="https://www.w3.org/community/solid/charter/">charter</a>, and <a href="https://github.com/w3c-cg/solid/blob/main/CONTRIBUTING.md">contributing guidelines</a>.</dd>
```
Comment (TallTed) [inline CONTRIBUTING.md:7]:
```suggestion
    <dd>This document is no longer in effect and is no longer being updated here. For the latest information on the <a href="https://www.w3.org/groups/cg/solid/">W3C Solid Community Group</a>, please refer to its <a href="https://github.com/w3c-cg/solid/">repository</a>, <a href="https://www.w3.org/community/solid/charter/">charter</a>, and <a href="https://github.com/w3c-cg/solid/blob/main/CONTRIBUTING.md">contributing guidelines</a>.</dd>
```

File: meetings/README.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
+<details open="">
+  <summary>Document Status</summary>
+  <dl>
+    <dt>Modified</dt>
+    <dd>2024-01-10</dd>
+    <dt>Reason</dt>
+    <dd>This document is no longer effective and is no longer being updated here. For the latest information on the <a href="https://www.w3.org/groups/cg/solid/">W3C Solid Community Group</a>, please refer to its <a href="https://github.com/w3c-cg/solid/">repository</a>, <a href="https://www.w3.org/community/solid/charter/">charter</a>, and <a href="https://github.com/w3c-cg/solid/blob/main/CONTRIBUTING.md">contributing guidelines</a>.</dd>
```
Comment (TallTed) [inline meetings/README.md:7]:
```suggestion
    <dd>This document is no longer in effect and is no longer being updated here. For the latest information on the <a href="https://www.w3.org/groups/cg/solid/">W3C Solid Community Group</a>, please refer to its <a href="https://github.com/w3c-cg/solid/">repository</a>, <a href="https://www.w3.org/community/solid/charter/">charter</a>, and <a href="https://github.com/w3c-cg/solid/blob/main/CONTRIBUTING.md">contributing guidelines</a>.</dd>
```

File: meetings/template.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
+<details open="">
+  <summary>Document Status</summary>
+  <dl>
+    <dt>Modified</dt>
+    <dd>2024-01-10</dd>
+    <dt>Reason</dt>
+    <dd>This document is no longer effective and is no longer being updated here. For the latest information on the <a href="https://www.w3.org/groups/cg/solid/">W3C Solid Community Group</a>, please refer to its <a href="https://github.com/w3c-cg/solid/">repository</a>, <a href="https://www.w3.org/community/solid/charter/">charter</a>, and <a href="https://github.com/w3c-cg/solid/blob/main/CONTRIBUTING.md">contributing guidelines</a>.</dd>
```
Comment (TallTed) [inline meetings/template.md:7]:
```suggestion
    <dd>This document is no longer in effect and is no longer being updated here. For the latest information on the <a href="https://www.w3.org/groups/cg/solid/">W3C Solid Community Group</a>, please refer to its <a href="https://github.com/w3c-cg/solid/">repository</a>, <a href="https://www.w3.org/community/solid/charter/">charter</a>, and <a href="https://github.com/w3c-cg/solid/blob/main/CONTRIBUTING.md">contributing guidelines</a>.</dd>
```

**Review by TallTed — CHANGES_REQUESTED** [review]
a few grammar fixes

**Comment by michielbdejong** [comment]
This PR is on the [agenda for the 17 January CG Call](https://hackmd.io/1YFPpDmnR-GwpjTGugVXcw)

**Review by elf-pavlik — APPROVED** [review]

**Review by VirginiaBalseiro — APPROVED** [review]

**Pull request merged**